src/osa/scripts/closer.py

```diff
@@ -169,6 +175,20 @@ def post_process(seq_tuple):
     if not options.no_dl2:
         merge_files(seq_list, data_level="DL2")
 
+    time.sleep(300)
+
+    # Check if all jobs launched by autocloser finished correctly 
+    # before creating the NightFinished.txt file
+    n_max = 10
+    n = 0
+    while not all_closer_jobs_finished_correctly() & n <= n_max:
```

Comment:
So this way would keep autocloser running in background up to 5*10 mins, right?
Some numbers on the time each job usually takes:
Provprocess: up to ~12 min
Muon merging: up to ~1 min
DL1 merging: up to ~30 min!
DL1 datacheck merging runwise: up to ~5 min
Datacheck merging daily: ~3 min

So maybe the check can be done with less frequency than 5 min

And how would you know whether some jobs were not finished properly? Checking the autocloser logs? I would try something like alerting via email if not all closer jobs are finished correctly after a few checks. What do you think?

Reply:
Ok, I will add an email alert to be sent if the jobs don't finish correctly. So you suggest making the check for example every 2 minutes? And up to 30 minutes?

Reply:
no, the other way around, maybe every 10-15 min up to ~1h?